step.yml

```diff
@@ -146,6 +146,12 @@ inputs:
       value_options:
         - "yes"
         - "no"
+  - env_var_lookup_keys:
+    opts:
+      title: "List of env var keys to lookup"
+      summary: "A list of env variable keys to lookup in order to determine whether the build should be cached or not."
+      description: "A list of env variable keys to lookup in order to determine whether the build should be cached or not."
+      is_expand: true
```

Comment:
Is expanding correct for this one?

Reply:
👍 fixed on b29e995b643cfb9730ca80d3dff7adada17554bb